README.md

```diff
@@ -37,9 +37,10 @@ $ docker run --log-driver=sumologic --log-opt sumo-url=https://<deployment>.sumo
 ### Sumo Logic Options
 To specify additional logging driver options, you can use the `--log-opt NAME=VALUE` flag.
 
-| Option                      | Required? | Default Value | Description
-| --------------------------- | :-------: | :-----------: | -------------------------------------- |
+| Option                    | Required? | Default Value | Description
+| ------------------------- | :-------: | :-----------: | -------------------------------------- |
 | sumo-url                  | Yes       |               | HTTP Source URL
+| sumo-source-category      | No        | `dockerlog` | Source category to appear when searching on Sumo Logic by `_sourceCategory`. To include the log tag in the source category, use `{{.Tag}}`; see `tag`.
```

Comment:
we should add the same support for `_sourceHost` and `_sourceName`

Reply:
should we allow the tag to be in `_sourceHost` and `_sourceName` as well?

Reply:
yep, basically 3 ways for customer to specify the metadata